lib/metasploit/credential/importer/core.rb

```diff
@@ -224,7 +224,7 @@ def import_short_form
       end
       if all_creds_valid
         core_opts.each do |item|
-          if Metasploit::Credential::Core.where(private:item[:private]).blank?
+          if Metasploit::Credential::Core.where(public:item[:public],private:item[:private]).blank?
```

Comment:
This should have some spaces in the hash to make it a bit clearer as to whats going on
`if Metasploit::Credential::Core.where(public: item[:public], private: item[:private]).blank?`